_inc/client/plans/single-product-backup.jsx

```diff
@@ -6,6 +6,35 @@ import { translate as __ } from 'i18n-calypso';
 
 import './single-product-backup.scss';
 
+export function PlanPriceDisplay( { backupPlanPrices, currencySymbol } ) {
+	const perYearPriceRange = `${ currencySymbol }${ backupPlanPrices.jetpack_backup_daily.yearly }-${ backupPlanPrices.jetpack_backup_realtime.yearly } /year`;
+
+	const full_daily_backup_yearly_cost = backupPlanPrices.jetpack_backup_daily.monthly * 12;
+	const full_realtime_backup_yearly_cost = backupPlanPrices.jetpack_backup_realtime.monthly * 12;
+
+	return (
+		<div className="single-product-backup__plan-price-display-container">
```

Comment:
While this class name is long, it doesn't seem to be too descriptive. Those plan prices live in the single product backup header, so the class name like:

```scss
.single-product-backup__header-price
```
would be shorter and describe the element clearer in my opinion. What do you think?

Reply:
Agreed.